.env

```diff
@@ -74,3 +74,7 @@ ROUTEMASTER_MONITOR_INTERVAL=10
 ROUTEMASTER_SCALING_THRESHOLD=100
 ROUTEMASTER_SCALING_DEADLINE=1000
 
+# maximum number of bytes in an event's optional data payload (when messagepack'ed)
+# DO NOT INCREASE THIS - using the event bus as a data bus is a strong smell of
+# an incompatible architecture, and will cause major bus scalability issues.
+ROUTEMASTER_MAX_EVENT_DATA=64
```

Comment:
The rider location payload looks like this, which I suspect is more than 64 bytes, though I haven't tried it out:

```
_links:
  rider:
    href: https://deliveroo.co.uk/api/Internal/riders/18374
created_at: 2017-03-22T09:37:22.736Z
latitude: 52.84757364
longitude: -0.837465738
accuracy_m: 53.2
```

I can't see any reasonable way of reducing it, as it does need to link back to the rider. I'll try msgpack on it when I'm at a proper computer.